model/metrics/container.yaml

```diff
@@ -14,6 +14,20 @@ groups:
         brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
         requirement_level: opt_in
 
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
+    stability: experimental
+    brief: "CPU usage"
+    note: >
+      CPU usage of the specific container on all available CPU cores, averaged over the sample window
+    instrument: gauge
+    unit: "{cpu}"
+    attributes:
+      - ref: container.cpu.state
+        brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
```

Comment:
> A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."

Could it be clarified?
cpu state does not characterizes a container, but a state of the CPU. Also, does it mean that cpu.state should be added on all data points or not applied at all for a given container?